DynamicSmoothedGUI.py

```diff
@@ -382,9 +382,9 @@ def updateGridSmoothed(self):
                 lidar_data = self.generate_sensor.generateLidar(
                     degree_freq, curr_tile.row, curr_tile.col)
                 self.getPathSet()
-                self.recalc = self.gridEmpty.updateGridLidar(curr_tile.x, 
-                    curr_tile.y, lidar_data, robot_radius, bloat_factor, 
-                    self.pathSet, self.gridFull)
+                self.recalc = self.gridEmpty.update_grid_tup_data(curr_tile.x,
+                                                               curr_tile.y, lidar_data, 3, robot_radius, bloat_factor,
```

Comment:
Remember that the argument should be sensor type and it was just 3 for testing purposes